app/models/conference.rb

```diff
@@ -25,6 +25,22 @@ def when
     DateFormatter.new(start_date, end_date).to_s
   end
 
+  def cfp_status
+    if cfp_end_at
+      if cfp_end_at > Time.now
+        "CFP is open till #{pretty_cfp_end_at}"
+      else
+        "CFP has been closed on #{pretty_cfp_end_at}"
+      end
+    else
+      "CFP is open"
+    end
+  end
+
+  def pretty_cfp_end_at
+    cfp_end_at.strftime('%d %b %Y')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.